src/js/audio.js

```diff
@@ -1,10 +1,85 @@
+function eventListener(audio, ev) {
+  const progress = parseInt(100 * audio.audio.currentTime / audio.audioLength, 10);
+
+  // Dispatch progress event at around 10%, 25%, 50%, 75% and 100%
+  if (ev.type === 'timeupdate' && !shouldDispatch(progress)) {
+    return;
+  }
+
+  fireEvent(ev.type, audio);
+}
+
+function shouldDispatch(progress) {
+  const relevantProgressPoints = [8, 9, 10, 11, 12,
+                                  23, 24, 25, 26, 27,
+                                  48, 49, 50, 51, 52,
+                                  73, 74, 75, 76, 77,
+                                  100];
+  return relevantProgressPoints.includes(progress);
```

Comment:
If it's aligned with o-video then I'd keep it the same for now but we might need to review that at some point in the future